docs/diagrams/DeliveryCreateNoteSequenceDiagram.puml

```diff
@@ -0,0 +1,98 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":DeliveryCreateNoteCommandParser" as DeliveryCreateNoteCommandParser LOGIC_COLOR
+participant "d:DeliveryCreateNoteCommand" as DeliveryCreateNoteCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("delivery note 1 --note A note")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("delivery note 1 --note A note")
+activate AddressBookParser
+
+create DeliveryCreateNoteCommandParser
+AddressBookParser -> DeliveryCreateNoteCommandParser
+activate DeliveryCreateNoteCommandParser
+
+DeliveryCreateNoteCommandParser --> AddressBookParser
+deactivate DeliveryCreateNoteCommandParser
+
+AddressBookParser -> DeliveryCreateNoteCommandParser : parse("1 --note A note")
+activate DeliveryCreateNoteCommandParser
+
+create DeliveryCreateNoteCommand
+DeliveryCreateNoteCommandParser -> DeliveryCreateNoteCommand
+activate DeliveryCreateNoteCommand
+
+DeliveryCreateNoteCommand --> DeliveryCreateNoteCommandParser : d
+deactivate DeliveryCreateNoteCommand
+
+DeliveryCreateNoteCommandParser --> AddressBookParser : d
+deactivate DeliveryCreateNoteCommandParser
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+DeliveryCreateNoteCommandParser -[hidden]-> AddressBookParser
+destroy DeliveryCreateNoteCommandParser
+
+AddressBookParser --> LogicManager : d
+deactivate AddressBookParser
+
+AddressBookParser --> LogicManager : d
+deactivate AddressBookParser
```

Comment:
Is this a duplication?

Reply:
Yep, good catch, will update accordingly